ilert/README.md

```diff
@@ -1,41 +1,48 @@
 ## Overview
 
-The [iLert][1] integration sends Datadog alerts to iLert and seamlessly take actions on these alerts within the iLert platform.
+The [ilert][1] integration sends Datadog alerts to ilert and seamlessly takes actions on these alerts within the ilert platform.
 
-Integrate with iLert to:
+Integrate with ilert to:
 
 - Trigger and resolve incidents from Datadog
 - Tackle incidents and set up escalation policies as they occur
 - Set up a daily reminder of who is on-call
+- Export metrics from Datadog to ilert to provide information about the health of your services on ilert status pages
```

Comment:
Hi @DaTiMy, what metrics are being exported? How is ilert exporting them?

Reply:
In your documentation, I see that ilert asks customers to create and share Datadog API and App keys. 

We would require ilert to implement [OAuth](https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/) for this use case to avoid sharing App keys, which are tied to users and very broad. Instead there is a individual scope to query metrics that customer can agree to through an OAuth flow.

Reply:
We've decided to remove this out of the documentation until OAuth Flow has been implemented.